cadasta/gui/tools/wizard/project_creation_wizard.py

```diff
@@ -0,0 +1,190 @@
+# coding=utf-8
+"""
+Cadasta **Cadasta project creation Dialog.**
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+
+"""
+
+import logging
+
+from qgis.PyQt.QtGui import (
+    QDialog,
+)
+from qgis.PyQt.QtCore import pyqtSignature
+from qgis.gui import QgsMessageBar
+from step_project_creation01 import StepProjectCreation1
```

Comment:
Maybe you should rename it to `step_project_creation10`, `step_project_creation20` ... in case in the future we need to add an extra panel between 1 and 2, we never know ;-)
We have done this in inasafe : https://github.com/inasafe/inasafe/tree/develop/safe/gui/tools/wizard